content/03.results.md

```diff
@@ -8,6 +8,8 @@ Next, we created a Github analysis repository (https://github.com/AlexsLemonade/
 We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  
 
 Stub for the process for contributing analytical code (**Figure {@fig:FigS1}** and **STAR Methods**) and to the manuscript.
+
+#### Generation of consensus SNV and CNV calls
 Also add a quick note about consensus SNV and CNV to reference **Figure {@fig:FigS2}A-F** here.
```

Comment:
Should we comment this out, just to be safe?

Reply:
I took this out completely in the PR for figure 1, which I can't look up easily to link on my phone apparently. If we can get the open PRs reviewed, I can stagger any new PRs so this is easier to manage.

Reply:
took out here as well